common/web/gesture-recognizer/src/tools/unit-test-resources/src/hostFixtureLayoutController.ts

```diff
@@ -0,0 +1,91 @@
+/// <reference path="fixtureLayoutConfiguration.ts" />
+
+namespace Testing {
+  export class HostFixtureLayoutController extends EventEmitter {
+    public static readonly CONFIG_CHANGED_EVENT = "configchanged";
+
+    private _recognizer: com.keyman.osk.GestureRecognizer;
+    private _loadPromise: Promise<com.keyman.osk.GestureRecognizer>;
+
+    private _hostFixture: HTMLElement;
+    private _layoutConfig: FixtureLayoutConfiguration;
+
+    public constructor() {
+      super();
+      this._layoutConfig = new FixtureLayoutConfiguration();
+    }
+
+    public get recognizer() {
```

Comment:
What's the reason for not specifying the type here? Or why do you specify the type for `layoutConfiguration()` below?

Reply:
TS can pretty easily infer types in straightforward properties like these, though I'll still normally annotate them anyway just to be clear.  I might have gotten a bit lazy here and relied on that for this particular property.